generated/nidigital/nidigital/session.py

```diff
@@ -2134,13 +2636,25 @@ def get_time_set_edge(self, time_set_name, edge):
         nidigital.Session repeated capabilities container, and calling this method on the result.
 
         Args:
-            time_set_name (str):
+            time_set_name (str): The specified time set name.
+
+            edge (enums.TimeSetEdgeType): Name of the edge.
+
+                -   TimeSetEdgeType.DRIVE_ON (2800)
+                -   TimeSetEdgeType.DRIVE_DATA (2801)
+                -   TimeSetEdgeType.DRIVE_RETURN (2802)
+                -   TimeSetEdgeType.DRIVE_OFF (2803)
+                -   TimeSetEdgeType.COMPARE_STROBE (2804)
+                -   NIDIGITAL_VAL_DRIVE_DATA_2 (2805)
+                -   NIDIGITAL_VAL_DRIVE_RETURN_2 (2806)
+                -   NIDIGITAL_VAL_COMPARE_STROBE_2 (2807)
```

Comment:
The format should be updated to match the other enum values above (like TimeSetEdgeType.COMPARE_STROBE)

Reply:
It's not apparent to me why these aren't being handled already.

Reply:
OK, sort of figured it out.  The 'name' attribute in the metadata for these is missing an underscore before the '2' (e.g. `NIDIGITAL_VAL_DRIVE_DATA2`.  I'm looking into it.

Reply:
The correct spelling does not have '_' before the '2', so I've updated the documentation metadata.  The C API Reference documentation has the wrong spelling though.

Reply:
Filed an internal bug for C API reference docs (1276171)